tools/PI/DevHome.PI/Helpers/ExternalTool.cs

```diff
@@ -44,7 +44,19 @@ public string OtherArgs
     {
         get; private set;
     }
-
+
+    private bool isPinned;
+
+    public bool IsPinned
+    {
+        get => isPinned;
+        set
+        {
+            isPinned = value;
+            OnPropertyChanged(nameof(IsPinned));
+        }
+    }

```

Comment:
Use ObservableProperty (for other properties below as well)

```suggestion
    [ObservableProperty]
    private bool _isPinned;
```

Reply:
Fixed in 2nd commit.